Formula/sfml.rb

```diff
@@ -26,8 +26,15 @@ class Sfml < Formula
   # https://github.com/Homebrew/homebrew/issues/40301
   depends_on :macos => :lion
 
+  # install pkg-config files, currently skips OS X
+  patch :p0 do
+    url "https://gist.githubusercontent.com/llongi/0cb8b9ad00c858de05f22e839844f6e1/raw/0492d6f75b6da1c2e55993bd13bdd62667dbadd1/sfml_2.4.2_osx_homebrew_pkgconfig.patch"
```

Comment:
Let's just use an `inreplace` for this. Also the comment should have the url for the commit that it corresponds to upstream, the date of that commit, and for which version this can be removed.

Reply:
Done as requested. Uses inreplace now, comment mentions all information.